### PR TITLE
[1.0] Added more how-to examples

### PR DESCRIPTION
**Pull request by adnanrahic**
This PR adds another how-to example of sending syslog data to Sematext, which is an alternative for Splunk.

### CODE REVIEW

**Comment by adnanrahic** [comment]
Hey! This is my first contribution to Fluentd. Please let me know if I missed anything or if anything needs fixing. :smile:

File: guides/logs-to-sematext.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+and alerts to fine-tune your own personal use-case.
+
+If you're used to Kibana, you can still use it as well.
+For more details on how to use Kibana, please read [the official manual](https://www.elastic.co/guide/en/kibana/current/index.html).
+
+![](/images/sematext-logs-overview.png)
+
+### Debugging
+
+To manually send logs to Elasticsearch, please use the `logger` command.
```
Comment (otisg) [inline guides/logs-to-sematext.md:132]: to Elasticsearch or to Sematext?

File: guides/logs-to-sematext.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  index_name <LOGS_TOKEN>
+  <buffer>
+    @type file
+    path /tmp/fluent/es-buffer/es.all.*.buffer
+    chunk_limit_size 250k
+    flush_interval 50s
+    retry_limit 5
+    retry_wait 60
+  </buffer>
+  num_threads 1
```
Comment (repeatedly) [inline guides/logs-to-sematext.md:92]: use `flush_thread_count` in `<buffer>`. `num_threads` is for v0.12.

File: guides/logs-to-sematext.md (hunk 1 above)
Comment (adnanrahic) [reply guides/logs-to-sematext.md:132]: Aha, good catch. I'll change it.

File: guides/logs-to-sematext.md (hunk 2 above)
Comment (adnanrahic) [reply guides/logs-to-sematext.md:92]: Got it, will edit.

Comment (repeatedly) [reply guides/logs-to-sematext.md]: Please move it into `<buffer>` section.

```
<buffer>
  # other parameters...
  flush_thread_count 1
</buffer>
```

Comment (adnanrahic) [reply guides/logs-to-sematext.md]: @repeatedly fixed! Moved to under `<buffer>` section. :smile:

**Comment by repeatedly** [comment]
Thanks!

**Comment by adnanrahic** [comment]
My pleasure! Thanks for reviewing and merging so quickly. :smile:

**Pull request merged**